ophyd/epics_motor.py

```diff
@@ -42,12 +48,23 @@ class EpicsMotor(Device, PositionerBase):
     settle_time : float, optional
         The amount of time to wait after moves to report status completion
     '''
+    user_offset = Cpt(EpicsSignal, '.OFF')
     user_readback = Cpt(EpicsSignalRO, '.RBV')
     user_setpoint = Cpt(EpicsSignal, '.VAL', limits=True)
     motor_egu = Cpt(EpicsSignal, '.EGU')
     motor_is_moving = Cpt(EpicsSignalRO, '.MOVN')
     motor_done_move = Cpt(EpicsSignalRO, '.DMOV')
     motor_stop = Cpt(EpicsSignal, '.STOP')
+    motor_offset_freeze_switch = Cpt(EpicsSignal, '.FOFF')
```

Comment:
:+1: to remove the redundant 'motor_' . Just worried about the back-compat as @danielballan pointed.